.github/workflows/phpunit-oci.yml

```diff
@@ -8,16 +8,16 @@ name: PHPUnit
 on:
   pull_request:
     paths:
-      - '.github/workflows/**'
-      - 'appinfo/**'
-      - 'lib/**'
-      - 'templates/**'
-      - 'tests/**'
-      - 'vendor/**'
-      - 'vendor-bin/**'
-      - '.php-cs-fixer.dist.php'
-      - 'composer.json'
-      - 'composer.lock'
+      - ".github/workflows/**"
```

Comment:
Is this necessary, or can we stick with what the template does??
https://github.com/nextcloud/.github/blob/master/workflow-templates/phpunit-oci.yml#L11

Reply:
Sorry auto merge.
I just backported the PR of @kesselb.

Reply:
It should be enough to use the original version. 
I did not modify it on purpose but renamed the file from -yml to .yml.